meinberlin/apps/organisations/models.py

```diff
@@ -9,7 +9,14 @@
 
 
 class Organisation(models.Model):
-    slug = AutoSlugField(populate_from='name', unique=True)
+    slug = AutoSlugField(
+        populate_from='name',
+        unique=True,
+        editable=True,
+        help_text=_(
```

Comment:
So, now you can remove the help text again and redo the migration. :)

Reply:
Awesome:) Thank you! I will do this and remove the second commt and put it into a different PR